demo/components/radio-group-demo.tsx

```diff
@@ -0,0 +1,29 @@
+import React = require('react');
+import {RadioGroup, RadioButton} from '../../src';
+import {observable, action} from 'mobx';
+import {observer} from 'mobx-react';
+
+
+@observer
+export class RadioGroupDemo extends React.Component<{}, {}> {
+    @observable myValue: string = '';
+
+    @action
+    onChange = (value: string) => {
+        this.myValue = value;
+    };
+
+    render() {
+        return (
+            <div data-automation-id="RADIO_GROUP_DEMO">
+                <RadioGroup onChange={this.onChange}>
```

Comment:
And we should have an example of rendering a data source too. And maybe also custom component children.